### PR TITLE
Fixes for android 24 emulator

### PR DESCRIPTION
**Pull request by fjricci**

### CODE REVIEW

File: Support/Scripts/install-android-emulator.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ package_list=()
 
 # This is required due to dependency issues in the sdk. Hopefully this can be removed in future
 # versions of the sdk.
-if [ "${api_level}" -gt 23 ]; then
```
Comment (sas) [inline Support/Scripts/install-android-emulator.sh]: I'm not sure you should remove these quotes. If the argument is actually a numeric argument, it won't make a difference, but if the argument is not (error case), the quotes will prevent `[` from failing with stupid errors.

File: Support/Testing/Travis/before-script.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ if [[ "$TARGET" == Android-* && -n "${LLDB_TESTS-}" ]]; then
 
   case "${TARGET}" in
     "Android-ARM") emulator_arch="arm";;
-    "Android-ARM64") emulator_arch="aarch64";;
+    "Android-ARM64") emulator_arch="ranchu-arm64";;
```
Comment (sas) [inline Support/Testing/Travis/before-script.sh]: This is a little bit pathetic :D

File: Support/Scripts/install-android-emulator.sh (hunk 1 above)
Comment (fjricci) [inline Support/Scripts/install-android-emulator.sh]: I'm pretty sure this didn't work with the quotes.

Comment (sas) [inline Support/Scripts/install-android-emulator.sh]: I'm pretty sure language-wise it'll make no difference *unless* `${api_level}` contains something with spaces.

Comment (fjricci) [inline Support/Scripts/install-android-emulator.sh]: The language spec says you're right. I'll double check that it works tomorrow.

File: Support/Testing/Travis/before-script.sh (hunk 2 above)
Comment (xiaobai) [inline Support/Testing/Travis/before-script.sh]: Ranchu is the machine type used when invoking qemu-android-aarch64 or whatever it's called. I wish it was just `emulator64-arm64` with the default qemu argument being `-machine type=ranchu`. Including it in the name of the emulator binary feels strange. :/

**Pull request merged**